src/app/campaign/store/effects/campaign.effects.ts

```diff
@@ -61,7 +61,7 @@ export class CampaignEffects {
             deletedFlightDocs: { [id: number]: HalDoc };
           }) => {
             const createdFlights: { [id: number]: Observable<HalDoc> } = payload.createdFlights.reduce(
-              (acc, flight) => ({ ...acc, [flight.id]: this.campaignService.createFlight(flight) }),
```

Comment:
Passing in the campaignDoc now to fix this nasty bug: https://github.com/PRX/grove.prx.org/pull/171/files#r408448986